sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0-preview.1 (UNDER CONSTRUCTION)
+
+- Begin using vrsion 3.1-preview.1 that has support for opinion mining.
+- [Breaking] Renamed the models `TextDocumentStatistics` to `DocumentStatistics`, `SentimentConfidenceScores` to `SentimentConfidenceScorePerLabel`, and `TextDocumentBatchStatistics` to `RequestStatistics`.
```

Comment:
Are these renamed intended? Looking at `swagger/readme.md` it looks like these should have been overridden. If those overrides are not working let me know and I'll look into the generator

Reply:
The problem was in the transform itself, it is missing `transform: >` see my comment below, fixing swagger/readme.md should remove these renames